app/src/main/java/org/fossasia/openevent/general/Converter.kt

```diff
@@ -0,0 +1,21 @@
+package org.fossasia.openevent.general
+
+import android.arch.persistence.room.TypeConverter
+import com.google.gson.Gson
+import org.fossasia.openevent.general.event.Event
+
+class Converter {
+
+    @TypeConverter
+    fun stringToEvent(data: String?): Event? {
+        if (data == null) {
+            return null
+        }
+        return Gson().fromJson(data, Event::class.java)
+    }
```

Comment:
Save id of Event in int format, don't serialze and deserialize event since we are only loading event ID